<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogWidget</class>
 <widget class="QWidget" name="LogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>操作系统课程平台</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <spacer name="verticalTopSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Microsoft YaHei</family>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="text">
      <string>操作系统课程原理展示平台</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalMiddleSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="centerLayout">
     <item>
      <spacer name="horizontalLeftSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="formLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="usernameLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="accountLable">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>用户名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="accountLineEdit">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="passwordLayout">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="passLable">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>密　码：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passLineEdit">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="displayRadioButton">
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>显示</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <item>
          <widget class="QPushButton" name="loginButton">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>登录</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="enrollButton">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>注册</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteButton">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>注销</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelButton">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>35</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Microsoft YaHei</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>取消</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalRightSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalBottomSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
